ptbcontrib/username_to_chat_api/usernametochatapi.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""
+This module contains a class that, once initiated, works as a shortcut to the UsernameToChatAPI
+and puts the response in a Chat object, as well as puts the error to the fitting TelegramErrors.
+"""
+
+try:
+    import telegram.vendor.ptb_urllib3.urllib3 as urllib3
+except ImportError:  # pragma: no cover
+    import urllib3
+
+try:
+    import ujson as json
+except ImportError:  # pragma: no cover
+    import json  # type: ignore[no-redef]
+
+from telegram import error, Chat, Bot
+
+
+class UsernameToChatAPI:  # pylint: disable=too-few-public-methods
+    """
+    This class stores the URL and api key for a UsernameToChatAPI instance. Then, the correct
```

Comment:
initialization doesn't return anything, though?

Reply:
don't tell me, tell them.

You said ignore the errors, so that's what I told pylint to do.

Reply:
Ohh, I see.